### ISSUE TITLE
open-vm-tools fails to compile on nixos-unstable

### ISSUE THREAD

**Issue by peterhoeg** [NixOS/nixpkgs#14436]
## Issue description

open-vm-tools fails to build on nixos-unstable

Fails with a lot of ```'Bool' does not name a type``` messages:

```
/tmp/nix-build-open-vm-tools-9.10.0-2476743.drv-0/open-vm-tools-9.10.0-2476743/lib/include/unicodeTypes.h:476:1: error: 'Bool' does not name a type
```

### Steps to reproduce

1. Set ```services.vmwareGuest.enable = true;```
2. ```nixos-rebuild test```

## Technical details

* System: 16.09pre79453.32b7b00 (Flounder)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: "16.09pre79453.32b7b00"

**Comment by domenkozar** [comment]
Due to gtkmm being bumped, we also need to bump open-vm-tools: https://github.com/vmware/open-vm-tools/releases

**Comment by peterhoeg** [comment]
I have actually tried getting the latest 10.0.7 to compile, but it's currently blowing up and I didn't have time to look into it any further. Should we just depend on an older gtkmm in the mean time?

**Comment by puffnfresh** [comment]
Looks like this was fixed in #14911

**Comment by globin** [comment]
Confirmed it builds on master.
